linode/ipv6range/framework_resource.go

```diff
@@ -116,6 +116,11 @@ func (r *Resource) Create(
 		return
 	}
 
+	// If user configured linode_id, populate route_target implicitly.
+	if linodeIdConfigured {
+		data.RouteTarget = types.StringValue(ipv6range.RouteTarget)
+	}
+
```

Comment:
I think this part is not necessary because it will be populated in `FlattenIPv6Range` function, which is called below?

Reply:
We probably don't want `FlattenIPv6Range` to take care of RouteTarget, and we only want to compute it when linode_id is configured by user to create ipv6_range. If user create a ipv6_range using route_target, we won't need to compute it.

Reply:
Hi @yec-akamai, I think `FlattenIPv6Range` was designed to handle these cases automatically. So when user create a IPv6 Range using `route_target`, `FlattenIPv6Range` will preserve the user configured value (a known value) as `true` is passed as the `preserveKnown` parameter. If the user create a IPv6 Range via Linode ID, the `route_target` will become an unknown value and will be overridden in `FlattenIPv6Range`.

Reply:
Ah okay, I see the problem there... `route_target` won't be in the response of get IPv6 range API call